src/App.tsx

```diff
@@ -1,15 +1,20 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
 
 export const App: React.FC = () => {
+  const [query, setQuery] = useState('');
+  const visibleMovies = moviesFromServer.filter(
+    movie => movie.title.toLowerCase().includes(query.toLowerCase())
+    || movie.description.toLowerCase().includes(query.toLowerCase()),
```

Comment:
In this line you duplicate you code, try to create and use a helper function for this purpose.

[checklist point 2](https://github.com/mate-academy/react_movies-list-filter/blob/master/checklist.md)